vmdb/lib/workers/schedule_worker.rb

```diff
@@ -233,6 +216,23 @@ def schedules_for_scheduler_role
       @queue.enq :storage_scan_timer
     end
 
+    schedules_for_ems_refresh
+  end
+
+  def schedules_for_ems_refresh
+    config = VMDB::Config.new("vmdb").config.fetch(:ems_refresh, {})
+
+    ExtManagementSystem.leaf_subclasses.each do |klass|
+      every   = config.fetch_path(klass.ems_type.to_sym, :refresh_interval)
+      every ||= config.fetch(:refresh_interval, 24.hours)
```

Comment:
I don't see a key :refresh_interval in the base configuration @Fryguy?  What is this line pulling from?